<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowminimumheight="0,0,0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLanguage">
        <property name="text">
         <string>Select translation preference:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelLookup">
        <property name="text">
         <string>Look up:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="bKnow">
        <property name="text">
         <string>I know this word (Space)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="bUndo">
        <property name="text">
         <string>Undo (Ctrl+Z)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Newword</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Frequency</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextBrowser" name="textBrowser"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout" stretch="0,0,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="commandLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="bRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bTranslate">
          <property name="text">
           <string>Translate (Ctrl+T)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bGoogle">
          <property name="text">
           <string>Google (Ctrl+G)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bPaste">
          <property name="text">
           <string>Paste from Clipboard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bRemoveInFile">
          <property name="text">
           <string>Remove a word from memory (pro)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bSave">
          <property name="text">
           <string>Save analyse (pro)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bLoad">
          <property name="text">
           <string>Load virtual memory (pro)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>178</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Please open a reading file to begin</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Load virtual memory to begin</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_LR"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionAdd_a_dictionary"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout_LR">
   <property name="text">
    <string>About LR</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save to LR file</string>
   </property>
  </action>
  <action name="actionAdd_a_dictionary">
   <property name="text">
    <string>Add a dictionary</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
